extension/src/main/webapp/WEB-INF/conf/chat/templates/notification/push/ChatPushPlugin.gtmpl

```diff
@@ -1 +1,17 @@
-<%=_ctx.appRes("exoplatform.chat.MentionPlugin", USER, ROOM_NAME)%>
\ No newline at end of file
+<%
+def notificationDescription = "";
+switch(NOTIF_TYPE) {
+ case "ChatMentionNotificationPlugin":
+   notificationDescription =  _ctx.appRes("exoplatform.chat.MentionPlugin", USER, ROOM_NAME);
+   break;
+ default:
+    if(Boolean.parseBoolean(IS_GROUP_CHAT))
```

Comment:
Shoulmdn't we add a message like : "You have received a message in room ROOM_NAME" ?